Makefile

```diff
@@ -300,8 +308,27 @@ test:
 		TEST_TOKEN_UNALLOWED="$(shell cat $(BUILD_FOLDER)/auth-tokenUnallowedString)" \
 		ENABLE_AUTH="true" \
 		WITH_AMS_SUBSCRIPTIONS="true" \
+		ENABLE_KUBE_API="false" \
 		go test -v ./subsystem/... -count=1 $(GINKGO_FOCUS_FLAG) -ginkgo.v -timeout 120m
 
+enable-kube-api-for-subsystem: $(BUILD_FOLDER)
+	skipper make deploy-service-requirements ENABLE_KUBE_API=true ENABLE_AUTH=false
```

Comment:
you need to pass all the parameters to this command, profile, target and namespace

Reply:
change `ENABLE_AUTH` to `AUTH_TYPE` here too

Reply:
already did